code/modules/projectiles/guns/projectile/boltgun/armstrong.dm

```diff
@@ -4,19 +4,39 @@
 	icon = 'icons/obj/guns/projectile/lever.dmi'
 	icon_state = "lever"
 	item_state = "lever"
+	w_class = ITEM_SIZE_BULKY
 	slot_flags = SLOT_BELT|SLOT_BACK
 	force = WEAPON_FORCE_PAINFUL
 	caliber = CAL_MAGNUM
 	max_shells = 11
-	price_tag = 650
+	price_tag = 600
 	init_recoil = HMG_RECOIL(0.4)
 	damage_multiplier = 1.2
 	penetration_multiplier  = 1.1
+	zoom_factor = 0.4
```

Comment:
if this has a built in scope it prob shouldnt be able to take scopes

Reply:
Fixed